pom.xml

```diff
@@ -7,8 +7,6 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.target>1.8</maven.compiler.target>
-		<maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
<maven.compiler.target>11</maven.compiler.target>
<maven.compiler.source>11</maven.compiler.source>

Reply:
tried that. doesn't work for me currently. maybe I'm doing something wrong?
I removed maven-compiler-plugin and set 

<maven.compiler.target>11</maven.compiler.target>
<maven.compiler.source>11</maven.compiler.source>

in pom.

Then doing a Maven, Force update in eclipse. I got compile errors in most classes, * is not allowed for source level below 1.7. Seems it doesn't recognized compiler target.